lib/mix/tasks/dogma.ex

```diff
@@ -11,8 +11,11 @@ defmodule Mix.Tasks.Dogma do
     {dir, formatter} = argv |> parse_args
     config = Config.build
 
+    {:ok, events} = GenEvent.start_link([])
```

Comment:
Yes, as it stands right now if an event handler fails then the entire app crashes. This could be for something stupid, like not having a default `handle_event/2` function, so we should watch out for that with event handlers.